library/src/main/java/com/rockerhieu/emojicon/EmojiconGridFragment.java

```diff
@@ -66,8 +67,7 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             mData = People.DATA;
             mUseSystemDefault = false;
         } else {
-            Object[] o = (Object[]) getArguments().getSerializable("emojicons");
-            mData = Arrays.asList(o).toArray(new Emojicon[o.length]);
+            mData = (Emojicon[]) getArguments().getParcelableArray(EMOJICONS_KEY);
```

Comment:
I can reproduce it on my phone when I enable 'Don't keep activities' in the developer options. I can confirm that your pull request fixes it.

I got this stacktrace (maybe it has to do with the support library):

E/AndroidRuntime(26523): Caused by: java.lang.ClassCastException: android.os.Parcelable[] cannot be cast to com.rockerhieu.emojicon.emoji.Emojicon[]
E/AndroidRuntime(26523): 	at com.rockerhieu.emojicon.EmojiconGridFragment.onViewCreated(EmojiconGridFragment.java:70)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1035)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1197)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1179)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:1991)
E/AndroidRuntime(26523): 	at android.support.v4.app.Fragment.performActivityCreated(Fragment.java:1976)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1041)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1197)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1179)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:1991)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentController.dispatchActivityCreated(FragmentController.java:165)
E/AndroidRuntime(26523): 	at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:507)